LyndaDecryptor/Program.cs

```diff
@@ -46,7 +46,7 @@ static void Main(string[] args)
                     
 
                 if (decryptorOptions.UsageMode == Mode.Folder)
-                    decryptor.DecryptAll(decryptorOptions.InputPath, decryptorOptions.OutputPath);
+                    decryptor.DecryptAll(decryptorOptions.InputPath, decryptorOptions.OutputFolder);

```

Comment:
Enjoy!
Also check the other change in Decryptor.cs. In case you don't give an output directory (using a /OUT parameter) and you give a database (using a /DB) it will go, search through the database, get the correct video name but will throw an error "Could not retrive media information from database...Falling back to default behaviour" because the outputDirectory is null/not set to an instance of an object.
I set it to the same directory as the input one by default if none is received, it works just fine like this and it creates all the folder and nice video file names structure in the same directory as the input files. Maybe another fix would also be nice here, if the /OUT parameter does not give a legal file path do the same thing and rename the files created in the input folder.